dandiapi/api/views/asset.py

```diff
@@ -136,7 +136,7 @@ class Meta:
 
 
 class AssetViewSet(DetailSerializerMixin, GenericViewSet):
-    queryset = Asset.objects.all().order_by('created')
+    queryset = Asset.objects.all().select_related('zarr').order_by('created')
```

Comment:
how does this change relate to https://github.com/dandi/dandi-archive/pull/1216/files#diff-00bb976899730dcae9a407d74d487ec5ba609713007bb4bad93ca68da650cc1cR522 mentioned above -- isn't it also for the "listing of assets"?

Reply:
different view. but indeed shows an alternative approach of "take all 3 since we know they would come useful"